src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -70,7 +70,9 @@ limitations under the License.
             <include cluster="Operational Credentials" client="false" server="true" clientLocked="true" serverLocked="true"></include>
             <include cluster="Localization Configuration" client="false" server="false" clientLocked="true" serverLocked="false"></include>
             <include cluster="Localization Time Format" client="false" server="false" clientLocked="true" serverLocked="false"></include>
-            <include cluster="Localization Unit" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Localization Unit" client="false" server="false" clientLocked="true" serverLocked="false">
```

Comment:
There's no such cluster... it's called "Unit Localization".

Similar for other clusters here: the names are off.

Reply:
Thanks for catching that